ceph/cephV2/osd.py

```diff
@@ -0,0 +1,19 @@
+"""Module with CLI wrapper for `ceph osd ` commands
+
+This module is to contain CLI wrapper for all `ceph osd ` commands
+and modules for sub-commands of `ceph osd ` would use this module.
+"""
+
+from ceph.cephV2.ceph import Ceph
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Osd(Ceph):
```

Comment:
use composition inheritance

Reply:
Have migrated all classes to composite inheritance
Please review